packages/core/src/percy.js

```diff
@@ -360,6 +360,11 @@ export class Percy {
       }
     }.call(this));
   }
+
+  shouldSkipAssetDiscovery(tokenType) {
+    if (this.testing && JSON.stringify(this.testing) === JSON.stringify({})) { return true; }
```

Comment:
* is `testing` ever an empty `{}` dictionary? - that's weird

Reply:
Yes refer to following here, not changing behaviour as existing flow will need change and tests fails, we can change as part of later though.
https://github.com/percy/cli/blob/9c89383d6e1a9aa41f0c83f8e4b9ddcdf827583b/packages/core/src/percy.js#L83